target/linux/ramips/dts/mt7628an_xiaomi_mi-ra75.dts

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an_xiaomi_mi-router-4.dtsi"
+
+/ {
+	compatible = "xiaomi,mi-ra75", "mediatek,mt7628an-soc";
+	model = "Xiaomi Mi AC1200 WLAN Range Extender RA75";
+        aliases {
+                led-boot = &led_system_orange;
+                led-failsafe = &led_system_orange;
+                led-running = &led_system_blue;
+                led-upgrade = &led_system_orange;
+        };
+
+        leds {
+                compatible = "gpio-leds";
+
+                led_signal_blue: signal_blue {
+                        label = "blue:signal";
+                        gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+                };
+                led_signal_orange: signal_orange {
+                        label = "orange:signal";
+                        gpios = <&gpio 37 GPIO_ACTIVE_LOW>;
+                };
+                led_system_blue: system_blue {
+                        label = "blue:system";
+                        gpios = <&gpio 64 GPIO_ACTIVE_LOW>;
+                };
+                led_system_orange: system_orange {
+                        label = "orange:system";
+                        gpios = <&gpio 66 GPIO_ACTIVE_LOW>;
+                };
+        };
+
+        keys {
+                compatible = "gpio-keys";
+
+                reset {
+                        label = "reset";
+                        gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+                        linux,code = <KEY_RESTART>;
+                };
+                wps {
+                        label = "wps";
+                        gpios = <&gpio 67 GPIO_ACTIVE_LOW>;
+                        linux,code = <KEY_WPS_BUTTON>;
+                };
+        };
+
+};
+
+&partitions {
+	partition@60000 {
+		label = "overlay";
+		reg = <0x60000 0x100000>;
+		read-only;
+	};
+
+	partition@160000 {
+		label = "firmware";
+		reg = <0x160000 0xea0000>;
+		compatible = "denx,uimage";
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie0 {
+	wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0000 0 0 0 0>;
+		mediatek,mtd-eeprom = <&factory 0x8000>;
+		ieee80211-freq-limit = <5000000 6000000>;
+	};
+};
+
+&ethernet {
+	nvmem-cells = <&macaddr_factory_4>;
+	nvmem-cell-names = "mac-address";
+	mac-address-increment = <(-1)>;
+};
+
+&esw {
+	mediatek,portmap = <0x3e>;
+	mediatek,portdisable = <0x2a>;
+};
+
+&factory {
+	compatible = "nvmem-cells";
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	macaddr_factory_4: macaddr@4 {
+		reg = <0x4 0x6>;
+	};
```

Comment:
I think this should also be added:
```
	macaddr_factory_28: macaddr@28 {
		reg = <0x28 0x6>;
	};
	macaddr_factory_8004: macaddr@8004 {
		reg = <0x8004 0x6>;
	};
```
In my `factory` dump, I see the following MAC addresses:
```
0x4:    3C:CD:57:7E:A0:78
0x28:   64:64:4A:FD:2E:B3
0x8004: 3C:CD:57:7E:A0:79  
```

Reply:
OK, I'll check with a fresh unit if these are actually used with the stock firmware, and for what. Adding them as suggested won't use them, right? 
BTW, How did you get your factory dump? Hardware adapter?

Reply:
> BTW, How did you get your factory dump? Hardware adapter?

A full dump was thrown to me by a user from the `4pda` forum. Where he got it from I don't know.